data/hai/hai reveal 5 philosophers.txt

```diff
@@ -626,7 +626,7 @@ mission "Hai Reveal [B08] Syndicate Internal Affairs"
 		has "Hai Reveal [B07-1] Boarded Smuggler: done"
 	on offer
 		fail "Hai Reveal [B02-S] Republic Intelligence Stalkers"
-		log `Giti tripped an alert which places you both under the jurisdiction of a Syndicate Internal Affairs investigation. You've were sent directly to Hephaestus.`
+		log `Giti tripped an alert which places you both under the jurisdiction of a Syndicate Internal Affairs investigation. You've been told to go directly to Hephaestus.`
```

Comment:
Logs are written in a personless form, not referring to the player in the first, second, or third person.
```suggestion
		log `Have been placed under investigation by Syndicate Internal Affairs alongside Giti after she tripped an alert.. Ordered to go directly to Hephaestus.`
```